vsintegration/src/FSharp.Editor/Classification/ClassificationService.fs

```diff
@@ -139,7 +139,9 @@ type internal FSharpClassificationService [<ImportingConstructor>] () =
 
                 let! cancellationToken = CancellableTask.getCurrentCancellationToken ()
 
-                let defines, langVersion = document.GetFSharpQuickDefinesAndLangVersion()
+                let defines, langVersion, strictIndentation =
+                    document.GetFSharpQuickDefinesAndLangVersion()
```

Comment:
Rename to something like "document.GetFsharpParsingOptions" to match what it does and returns?